app/models/user.rb

```diff
@@ -98,9 +98,9 @@ def available_groups(order)
   def self.find_for_google_oauth2(access_token, _signed_in_resource = nil)
     data = access_token.info
     user = find_or_initialize_by(email: data.email) do |u|
-      u.name = data.name
       u.password = Devise.friendly_token[0, 20]
```

Comment:
what happens to the previous password in a case where the user exists?

Reply:
The block is only called to initialize a new user if the user does not exist.

Reply:
Actually, I know I just approved the PR, but can you confirm why you moved the logic for setting u.name outside of the initialize block?

Reply:
A user is created with the email but without a name when an invitation is sent, so the name needs to be set for that user. It won't update if a name is present, so existing functionality is preserved.